script/rename.js

```diff
@@ -3,81 +3,125 @@
-                    '<input type="text" name="id" style="width:100%">' +
-                    '</label>' +
-                    '</form>' +
-                    '</div>'
-            );
-            $dialog.find('input[name=id]').val(JSINFO.id);
-            $dialog.find('form').submit(renameFN);
+        return false;
+    };
 
-            // set up the dialog
-            $dialog.dialog({
-                title: LANG.plugins.move.rename+' '+JSINFO.id,
-                width: 800,
-                height: 180,
-                dialogClass: 'plugin_move_dialog',
-                modal: true,
-                buttons: [
-                    {
-                        text: LANG.plugins.move.cancel,
-                        click: function () {
-                            $dialog.dialog("close");
-                        }
-                    },
-                    {
-                        text: LANG.plugins.move.rename,
-                        click: renameFN
+    /**
+     * Create the actual dialog modal and show it
+     */
+    const showDialog = function () {
+        $dialog.dialog({
+            title: LANG.plugins.move.rename + ' ' + JSINFO.id,
+            width: 800,
+            height: 200,
+            dialogClass: 'plugin_move_dialog',
+            modal: true,
+            buttons: [
+                {
+                    text: LANG.plugins.move.cancel,
+                    click: function () {
+                        $dialog.dialog("close");
                     }
-                ],
-                // remove HTML from DOM again
-                close: function () {
-                    jQuery(this).remove();
+                },
+                {
+                    text: LANG.plugins.move.rename,
+                    click: renameFN
                 }
-            })
+            ],
+            // remove HTML from DOM again
+            close: function () {
+                jQuery(this).remove();
+            }
         });
-}
+        $dialog.find('input[name=id]').val(JSINFO.id);
+        $dialog.find('form').submit(renameFN);
+    };
+
+    /**
+     * Bind an event handler as the first handler
+     *
+     * @param {jQuery} $owner
+     * @param {string} event
+     * @param {function} handler
+     * @link https://stackoverflow.com/a/4700103
+     */
+    const bindFirst = function ($owner, event, handler) {
+        $owner.unbind(event, handler);
+        $owner.bind(event, handler);
+
+        const events = jQuery._data($owner[0])['events'][event];
+        events.unshift(events.pop());
+
+        jQuery._data($owner[0])['events'][event] = events;
+    };
+
+
+    // attach handler to menu item
+    jQuery('.plugin_move_page')
+        .show()
+        .click(function (e) {
+            e.preventDefault();
+            showDialog();
+        });
+
+    // attach handler to mobile menu entry
+    $mobileMenuOption = jQuery('form select[name=do] option[value=plugin_move_page]');
```

Comment:
```suggestion
    const $mobileMenuOption = jQuery('form select[name=do] option[value=plugin_move_page]');
```
The variable should be declared.

Reply:
fixed